rootio/cmd/root-srv/main_go16.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !go1.7
+
+// root-srv runs a web server that can inspect and browse ROOT files.
+// root-srv can also display ROOT objects (TH1x, TH2x, TGraphs, TGraphErrors,
+// TGraphAsymmErrors, TDirectories, TTrees, ...).
+//
+// Usage: root-srv [options]
+//
+// ex:
+//
+//  $> root-srv -addr=:8080 &
+//  2017/04/06 15:13:59 server listening on :8080
+//
+//  $> open localhost:8080
+//
+//  $> root-srv -addr :8080 -serv https -host example.com
+//  2017/04/06 15:13:59 https server listening on :8080 at example.com
+
```

Comment:
please remove this empty line.
(otherwise the documentation won't be diplsayed by godoc)